add_me_to_your_PATH/sysimage.jl

```diff
@@ -18,11 +18,9 @@ Pkg.add("PackageCompiler")
 using PackageCompiler, UUIDs
 
 function main()
-    packages = [Symbol(k) for k in keys(project.dependencies)]
-    # exclude dependencies that are being developed
-    exclude = [Symbol(pi.name) for pi in values(Pkg.dependencies()) if !pi.is_tracking_registry]
+    packages = [Symbol(k) for k in keys(Pkg.project().dependencies)]
```

Comment:
Why was this line changed? Should the portion code before this be placed in this `main` function?

Reply:
Not sure why but if I put the `const project = ...` and following blocks in `main`, I get a

```
LoadError: expected the file `src/Example.jl` to exist for package `Example` at `/JuliaProject`
```.

I'm just reverting to using the `const project` in `main` as it was before.

Reply:
You cant define `const`ants in functions

Reply:
Oh I had removed the `const` keywords when I tried putting them in `main`.

Not sure what is causing the LoadError when I try to do in in `main`, but writing an empty `src/Example.jl` outside of `main` seems to do the trick.